### ISSUE TITLE
 Could not resolve './XXXX' from src\index.tsx

### ISSUE THREAD

**Issue by eightHundreds** [tanem/generator-typescript-react-lib#18]
```
// src/index2.tsx
const a=1
export default a
```

```
// src/index.tsx
import * as PropTypes from 'prop-types'
import * as React from 'react'
import a from './index2'

console.log(a)
// ....
```

```
// tsconfig.json
{
  	"noUnusedLocals": false,
    "noUnusedParameters": false,
}
```

```
npm run build
```

**Comment by tanem** [comment]
Just published `1.0.11` if you want to try the fix out?

**Comment by tanem** [comment]
Closing since this is ancient 😅
